Source/Mosa.Platform.x64/Intrinsic/CmpXChgLoad64.cs

```diff
@@ -18,7 +18,7 @@ private static void CmpXChgLoad64(Context context, MethodCompiler methodCompiler
 			var comparand = context.Operand3;
 			var result = context.Result;
 
-			var eax = Operand.CreateCPURegister(methodCompiler.TypeSystem.BuiltIn.U8, CPURegister.EAX);
+			var eax = Operand.CreateCPURegister(methodCompiler.TypeSystem.BuiltIn.U8, CPURegister.R1);
```

Comment:
Maybe this should use `RAX` specifically for both register name and variable name.